components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/SPInitLogoutRequestProcessor.java

```diff
@@ -165,6 +173,29 @@ public SAMLSSOReqValidationResponseDTO process(LogoutRequest logoutRequest, Stri
         }
     }
 
+    /**
+     * obtaining x509 Certificate for SAMLSSOServiceProviderDO for signature validation in a logout request
```

Comment:
If we are to add the java doc comment for even private methods we need to complete the comment. Otherwise we can avoid writing the doc comment for private methods.